tools/travis/splits/split_container.sh

```diff
@@ -51,6 +51,7 @@ run_test "Run kubernetes test" "$END_TO_END_DIR/test-scripts/test_kubernetes_emb
 run_test "Run kubernetes session test" "$END_TO_END_DIR/test-scripts/test_kubernetes_session.sh"
 if [[ "${HADOOP_INTEGRATION}" = "with-hadoop" ]]; then
     run_test "Run Mesos WordCount test" "$END_TO_END_DIR/test-scripts/test_mesos_wordcount.sh"
```

Comment:
After adding the new multiple submissions test, do we still need the word count test?

Reply:
I think we still need a "openbox" test case for deploying Flink on Mesos cluster. Anyway, it should be discussed in another ticket. WDYT?

Reply:
Can you define what you mean by _"openbox"_?

Reply:
I think each test case better not have multiple purposes so that we could find out the root cause quickly. I treat the WordCount case like a test for basic process of running Flink job on Mesos, including starting appmaster, running a simple job and verifying the correctness of result. I prefer to keep it in release 1.10, but I'm ok to remove it when the time overhead of E2E tests become severe.

Reply:
I agree that a separate test makes it easier to locate the root cause. However, due to being expensive, E2E tests should not be used for exhaustively testing all cases. Currently, running all E2E tests take more than 24 CPU hours on Travis. `e2e - container - hadoop 2.8` takes ~43 minutes on Travis, which is not great but acceptable. Imo, if we decide to add more Mesos tests, we should start to consolidate soon.

cc: @tillrohrmann

Reply:
I agree with your conclusion that ideally we try to separate test cases but in this case it might acceptable to consolidate them in order to save build time.

Reply:
Fine. So, in which release version we need to remove the WordCount case? 1.10 or 1.11? I prefer to remove it in next release version, when two other Mesos test  cases add. I don't have a strong opinion though.

Reply:
I am ok to consolidate them with the next test that we add.